apps/reference/README.md

```diff
@@ -43,3 +43,30 @@ We use documentation specifications which can be used to generate human-readable
 
 The benefit of using custom specifications is that we can generate many other types from a strict schema (eg, HTML and manpages).
 It also means that we can switch any documentation system we want. On this site we use Docusaurus, but in Supabase's official website we use a custom React site and expose only a subset of the available API for each tool.
+
+## Local setup
+
+To run the docs locally, you need to:
+
+- Clone the main [https://github.com/supabase/supabase](Supabase repo)
+  — Run `npm install` from root of the repo
+- Run `npm run dev` from the root of the repo, or from the `apps/reference` directory (if you want to run the reference docs only)
+- Access the docs from [http://localhost:3010/docs/](http://localhost:3010/docs/)
+
+## How to contribute?
+
+- Branch from `master` and name your branches with the following structure
+  - `{type}/{branch_name}`
+    - Type: `chore | fix | feature`
+    - The branch name is arbitrary — just make sure it summarizes the work.
+- When you send a PR to `master`, tag [https://github.com/dannykng](@dannykng) for a review.
+
+### Which files to edit?
```

Comment:
I don't think we're quite there yet, but the **Edit this page** button at the bottom of each doc should be the source of truth. We can circle back and add this once we have it all cleaned up.